src/gen-ts.ts

```diff
@@ -447,6 +455,34 @@ function handleMixin(feature: analyzer.ElementMixin, root: ts.Document) {
   );
 };
 
+/**
+ * Mixins can automatically apply other mixins, indicated by the @appliesMixin
+ * annotation. However, since mixins may be applied transitively, to know the
```

Comment:
since mixins may be applied transitively -> since mixins may be composed of other mixins

Reply:
Done (similar).